PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeShippingMethod.kt

```diff
@@ -0,0 +1,58 @@
+package com.paypal.android.paypalnativepayments
+
+import com.paypal.checkout.order.Options
+
+/**
+ * Shipping method details for an order via the PayPal Native Checkout UI.
+ * If you want to show shipping options in the PayPal Native Paysheet,
+ * provide `purchase_units[].shipping.options` when creating an orderID with
+ * the [`orders/v2` API](https://developer.paypal.com/docs/api/orders/v2/#definition-purchase_unit)
+ * on your server. Otherwise, our Paysheet won't display any shipping options.
+ */
+data class PayPalNativeShippingMethod internal constructor(
+    /**
```

Comment:
```suggestion

    /**
```

Missing new lines throughout file